server/api/db/sqldb/db.py

```diff
@@ -2858,20 +2893,19 @@ def _calculate_schedules_counters(
                     ]
                 )
             )
+            .group_by(Schedule.project, Schedule.name)
             .all()
         )
 
         project_to_schedule_pending_jobs_count = collections.defaultdict(int)
         project_to_schedule_pending_workflows_count = collections.defaultdict(int)
 
         for result in schedules_pending_count_per_project:
-            if (
-                result[2].to_dict()["name"]
-                == mlrun_constants.MLRunInternalLabels.workflow
-            ):
-                project_to_schedule_pending_workflows_count[result[0]] += 1
-            elif result[2].to_dict()["value"] == "job":
-                project_to_schedule_pending_jobs_count[result[0]] += 1
+            project_name, schedule_name, kind = result
+            if kind == mlrun_constants.MLRunInternalLabels.workflow:
+                project_to_schedule_pending_workflows_count[project_name] += 1
+            elif kind == mlrun.common.schemas.ScheduleKinds.job.value:
```

Comment:
Since `ScheduleKinds` is `StrEnum`
```suggestion
            elif kind == mlrun.common.schemas.ScheduleKinds.job:
```